event-handlers/src/test/java/no/sikt/nva/nvi/events/evaluator/EvaluateNviCandidateHandlerTest.java

```diff
@@ -877,6 +837,115 @@ void shouldRejectCandidateWithOnlySwedishCountryCode() throws IOException {
       assertEquals(testScenario.expectedEvaluatedMessage(), messageBody);
     }
 
+    @Test
+    void shouldEvaluateCandidateInOpenPeriod() throws IOException {
+      // Given a publication that fulfills all criteria for NVI reporting
```

Comment:
Not a fan of comments in tests, but they describe well what we are testing :)

Reply:
I'm hoping to clean up the tests in general and maybe convert these to Cucumber tests, but temporary solutions... Needed to describe the scenario for my own sake when writing the tests, and figured I might as well leave them ;-)